sdk/computefleet/arm-computefleet/review/arm-computefleet-api-fleets.api.md

```diff
@@ -0,0 +1,82 @@
+## API Report File for "@azure/arm-computefleet"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { Client } from '@azure-rest/core-client';
+import { HttpResponse } from '@azure-rest/core-client';
+import { OperationOptions } from '@azure-rest/core-client';
+import { OperationState } from '@azure/core-lro';
+import { Paged } from '@azure/core-paging';
+import { PollerLike } from '@azure/core-lro';
+import { RawHttpHeaders } from '@azure/core-rest-pipeline';
+import { RequestParameters } from '@azure-rest/core-client';
+import { StreamableMethod } from '@azure-rest/core-client';
+
+// @public
+export function $delete(context: AzureFleetContext, subscriptionId: string, resourceGroupName: string, fleetName: string, options?: FleetsDeleteOptionalParams): PollerLike<OperationState<void>, void>;
+
+// @public (undocumented)
+export function _$deleteDeserialize(result: Delete202Response | Delete204Response | DeleteDefaultResponse | DeleteLogicalResponse): Promise<void>;
```

Comment:
why those internal function is exported ?

Reply:
if we enable hierarchyClient: true, codegen will generate these two lines in package.json:
https://github.com/Azure/azure-sdk-for-js/pull/30358/files#diff-1ec02e477c12623ca429d4e7914b76de2c4ece1c244198a308f0ddacbca7fd13R16-R17